plugins/snowflake/scripts/test_sample_and_estimate.py

```
@@ -0,0 +1,337 @@
+        issues = self._make_issues([("A", True, 30)])
+        sample, meta = stratified_sample(issues, 10, 10)
+        self.assertEqual(meta["human"]["total"], 0)
+        self.assertEqual(meta["bot"]["total"], 10)
+        self.assertEqual(len(sample), 10)
+
+    def test_lowercase_keys_handled(self):
+        issues = [
+            {"project_key": "X", "is_bot": False, "ISSUEKEY": "X-1"},
+            {"project_key": "X", "is_bot": True, "ISSUEKEY": "X-2"},
+        ]
+        sample, meta = stratified_sample(issues, 1, 1)
```

Ruff (0.15.10)

[warning] Unpacked variable `meta` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)